10.0/fuel-ui-fake: fake ui ci for fuel 10

rkhozinov opened the pull request.

Commit:
10.0/fuel-ui-fake: fake ui ci for fuel 10

ekhomyakova commented on servers/partner-ci/10.0/ui-fake-nigthly.yaml:
```diff
@@ -0,0 +1,23 @@
+       property-file: build_number
+       current-parameters: true
+       predefined-parameters: |
+         TEST_PREFIX=test_nics*
```
TEST_PATH (see issue)

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+      - text:
+          name: SPECIFIC_PROPS
+          default: |
+            FUEL_QA_TEST_GROUP=prepare_slaves_9;
```
fuel-qa is not needed

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+    name: 'predefined_parameters'
+    fuel_release: '10.0'
+    fuel_release_checker: '10'
+    job_name: 'ui'
```
ui-fake

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+    recreate_venv: false
+    kvm_use: false
+    update_master: true
+    enable_proposed: true
```
fuel-related parameters are not needed (except fuel-ui)

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+    fuel_qa_refs: ''
+    fuel_qa_branch: 'origin/stable/newton'
+    fuel_ui_branch: 'origin/stable/newton'
+    docker_branch: 'master'
```
fake-ui

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+    skip_vote_failed: false
+
+    testrail_project: "Mirantis OpenStack"
+    testrail_suite: "[10.0][PCE] Mechanism for providing additional properties for NICs and Nodes via plugin"
```
[10.0][Fuel] UI testing

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+    testgroup_value_type: default
+    testgroup_value: '{testgroup}'
+
+    testgroup:
```
- test_cluster_workflows
- test_deployment_history

naumvd95 replied:
doen

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+          choosing-strategy: default
+          skip-tag: true
+      - git:
+          url: 'http://github.com/rkhozinov/fuel-ui-tests-docker'
```
ekhomyakova

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+    <<: *test-parameters
+    scm:
+      - git:
+          url: 'http://git.openstack.org/openstack/fuel-qa'
```
fuel-web instead

naumvd95 replied:
done

naumvd95 committed:
10.0/fake-ui: configuring reporter,testgroups,repos

mk1: need to re-review

naumvd95 committed:
10.0/fake-ui: redefine wrapper to avoid issues[1],change art-path

[1]
http://jenkins-tpi.bud.mirantis.net:8080/view/10.0.ui-fake/job/10.0.ui-fake.test_cluster_workflows/12/console

naumvd95 committed:
10.0/fake:change repo

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+    reporter-name: '{fuel_release}.{job_name}.reporter'
+    nightly_timer: 'H 21 * * *'
+    nightly_disabled: false
+    force_erase: false
```
Seems force_erase and force_reuse will not do anything

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+    testrail_suite: "[10.0][Fuel] UI testing"
+    testrail_suite_desc: "Suite description"
+    testrail_milestone: "10.0"
+    testrail_plan: "${{TREP_TESTRAIL_MILESTONE}} iso #${{SNAPSHOTS_ID}}"
```
Change `iso #${{SNAPSHOTS_ID}}` to "ui" with timestamp, e.g. `ui 03/10/2017 18:30`

naumvd95 replied:
need to reconfigure reporting before

rkhozinov replied:
you could get that code from contrail jobs

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+          scm_submodule_disabled: false
+
+      - '{reporter-name}':
+          plugin_version: ''
```
Seems it is not needed

naumvd95 replied:
its stub for correct job-working

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+    parameters:
+      - common-fuel-parameters:
+          debug:                 '{debug}'
+          snapshots_id:           ''
```
Maybe snapshots_id, force_erase, force_reuse and fuel_release_checker are not needed

naumvd95 replied:
some of them removed, the other- marked as stubs

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+          '{testgroup_value_type}': '{obj:testgroup_value}'
+          description: "test-group name"
+      - string:
+          name: TEST_PREFIX
```
TEST_PATH?

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+          description: "test-group name"
+      - common-fuel-parameters:
+          debug:                 '{debug}'
+          snapshots_id:           ''
```
not needed?

naumvd95 replied:
as above

ekhomyakova commented on servers/partner-ci/10.0/ui-fake.yaml:
```diff
@@ -0,0 +1,234 @@
+          url: 'http://github.com/openstack/fuel-web'
+          basedir: docker/fuel-web
+          branches:
+            - $FUEL_QA_BRANCH
```
FUEL_WEB_BRANCH

naumvd95 replied:
done

naumvd95 replied:
done

ekhomyakova commented on servers/partner-ci/builders/ui.fake.test.sh:
```diff
@@ -0,0 +1,28 @@
+
+. "${VENV_PATH}/bin/activate"
+cd ${WORKSPACE}/docker/
+pip install --upgrade docker-compose
```
Is this under RECREATE_VENV condition? I suppose, it should be

naumvd95 replied:
done
